manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -309,8 +309,10 @@ func (na *cnmNetworkAllocator) DeallocateService(s *api.Service) error {
 
 // IsAllocated returns if the passed network has been allocated or not.
 func (na *cnmNetworkAllocator) IsAllocated(n *api.Network) bool {
-	_, ok := na.networks[n.ID]
-	return ok
+	if _, ok := na.networks[n.ID]; !ok {
```

Comment:
this does not look needed as a change

Reply:
yeah this isn't needed, this is the same as `return ok`.

Reply:
right right. I had added some more logic to check for allocation. I removed it and did not revert this.